experiments/dataSources/worldBank/WorldBankSources.xml

```diff
@@ -0,0 +1,268 @@
+			<fields>
+				<field id="id" label="Id" ispolymorphic="false" isnamepointing="false" displaytype="TEXT" precision="1" scale="0" accessible="true" filterable="true" ogdisplaytype="DOUBLE"/>
+				<field id="name" label="Name" ispolymorphic="false" isnamepointing="false" displaytype="STRING" accessible="true" filterable="true"/>
+				<field id="sourceNote" label="Source Note" ispolymorphic="false" isnamepointing="false" displaytype="STRING" accessible="true" filterable="true"/>
+				<field id="sourceOrganization" label="Source Organization" ispolymorphic="false" isnamepointing="false" displaytype="URL" accessible="true" filterable="true"/>
+			</fields>
+			<conditions>
+				<condition type="fieldvalue" value="" enclosevalueinquotes="true" sourcetype="param" sourceparam="id" state="filterableoff" inactive="true" name="id"/>
+			</conditions>
+			<actions/>
+		</model>
+		<model id="SpecificIndicator" query="false" createrowifnonefound="false" datasource="WorldBank" verb="GET" payloadcontenttype="JSON" endpoint="/countries/all/indicators/{{indicator}}?format=json" pathtocontent="1" limitmethod="simple" limit="200" simplerecordslimitparameter="per_page">
+			<fields>
+				<field id="id" label="Id" ispolymorphic="false" isnamepointing="false" displaytype="TEXT" precision="1" scale="0" accessible="true" filterable="true" ogdisplaytype="DOUBLE"/>
+				<field id="countryiso3code" label="Countryiso3code" ispolymorphic="false" isnamepointing="false" displaytype="STRING" accessible="true" filterable="true"/>
+				<field id="date" label="Date" ispolymorphic="false" isnamepointing="false" displaytype="DOUBLE" precision="4" scale="0" accessible="true" filterable="true"/>
+				<field id="decimal" label="Decimal" ispolymorphic="false" isnamepointing="false" displaytype="DOUBLE" precision="1" scale="0" accessible="true" filterable="true"/>
+				<field id="unit" label="Unit" ispolymorphic="false" isnamepointing="false" displaytype="DOUBLE" precision="0" scale="0" accessible="true" filterable="true"/>
+				<field id="value" label="Value" ispolymorphic="false" isnamepointing="false" displaytype="DOUBLE" accessible="true" filterable="true" ogdisplaytype="STRING" scale="2"/>
+				<field id="country.value" label="Value" ispolymorphic="false" isnamepointing="false" displaytype="STRING" accessible="true" filterable="true"/>
+			</fields>
+			<conditions>
+				<condition type="fieldvalue" value="" enclosevalueinquotes="true" sourcetype="param" sourceparam="indicator" state="filterableoff" inactive="true" name="indicator"/>
+			</conditions>
+			<actions/>
+		</model>
+	</models>
+	<components>
+		<skuid__wrapper uniqueid="sk-2p5--1954">
+			<components>
+				<skuid__wizard uniqueid="sk-2pOG-6704" isResponsive="false">
```

Comment:
```suggestion
				<skuid__wizard uniqueid="sk-2pOG-6704" isResponsive="false" disableStepButtons="true">
```
Don't let users proceed in this wizard until they have queried data.

Because of that, we should also consider removing the `Next` buttons on the wizard also. But leave the `Back` buttons.

Reply:
Sure..